crates/bevy_core_pipeline/src/msaa_writeback.rs

```diff
@@ -81,13 +87,17 @@ impl Node for MsaaWritebackNode {
 
             let pass_descriptor = RenderPassDescriptor {
                 label: Some("msaa_writeback"),
-                // The target's "resolve target" is the "destination" in post_process
+                // The target's "resolve target" is the "destination" in post_process.
                 // We will indirectly write the results to the "destination" using
                 // the MSAA resolve step.
-                color_attachments: &[Some(target.get_color_attachment(Operations {
-                    load: LoadOp::Clear(Default::default()),
-                    store: StoreOp::Store,
-                }))],
+                color_attachments: &[Some(RenderPassColorAttachment {
+                    view: target.sampled_main_texture_view().unwrap(),
```

Comment:
It wasn't immediately obvious why this `.unwrap()` is fine so something like this could help a bit I think. Especially if someone ever changes the if you added in the future.

```suggestion
                    view: target.sampled_main_texture_view().expect("Sampled texture should exist when MSAA is enabled"),
```

Reply:
Added a comment